stdlib/tkinter/__init__.pyi

```diff
@@ -2674,11 +2676,11 @@ class Scrollbar(Widget):
     def configure(self, cnf: str) -> tuple[str, str, str, Any, Any]: ...
     config = configure
     def activate(self, index: Any | None = ...): ...
-    def delta(self, deltax, deltay): ...
-    def fraction(self, x, y): ...
-    def identify(self, x, y): ...
-    def get(self): ...
-    def set(self, first, last): ...
+    def delta(self, deltax: int | str, deltay: int | str) -> float: ...
+    def fraction(self, x: int | str, y: int | str) -> float: ...
+    def identify(self, x: int | str, y: int | str) -> Literal["arrow1", "arrow2", "slider", "."]: ...
+    def get(self) -> tuple[float, float, float, float] | tuple[float, float]: ...
+    def set(self, first: float | str, last: float | str) -> None: ...
```

Comment:
- These shouldn't allow strings
- `identify` should have `""` instead of `"."`
- Can `get()` return a tuple of only two floats? It seems to always return 4 floats to me.

Reply:
Been a little while since I added these ones, but I believe that I was able to make it return a 2-float tuple during some testing.

Reply:
Ok. I confirmed that it can return a 2 or 4 element tuple of floats.
sample code for `tuple[float, float, float, float]`

```python3
import tkinter as tk
sb = tk.Scrollbar()
print(sb.get())
```

Sample code for `tuple[float, float]`. Press the button to print the output. 

```python3
import tkinter as tk

root = tk.Tk()
text = tk.Text(root)

scrollbar = tk.Scrollbar(root)
button = tk.Button(root, text="Print Scrollbar", command=lambda: print(scrollbar.get()))

scrollbar.config(command=text.yview)
text.config(yscrollcommand=scrollbar.set)

text.grid(row=0, column=0, sticky=tk.NSEW)
scrollbar.grid(row=0, column=1, sticky=tk.NSEW)
button.grid(row=1, column=0, columnspan=2, sticky=tk.NSEW)
root.mainloop()
```